src/twisted/python/test/test_url.py

```diff
@@ -5,12 +5,18 @@
 """
 Tests for L{twisted.python.url}.
 """
-
 from __future__ import unicode_literals
 
-from ..url import URL
+import warnings
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", category=DeprecationWarning)
+    from ..url import URL
```

Comment:
OK @adiroiban pointed out in `#twisted-dev` that `deprecatedModuleAttribute` is _supposed_ to emit a deprecation warning once and only once, when the module's attribute is used.  You would think this would trigger the deprecation warning and break [the deprecation test](https://github.com/twisted/twisted/pull/776/files#diff-19a383d67f518cb991c253b231d176eeR807). However, it doesn't!  Neither of these do:

```python
from ..url import URL
from twisted.python.url import URL
```

But this does:

```
from twisted.python import url
URL = url.URL
```

Which leads me to believe that the import system's attribute access mechanism isn't just `getattr` (or perhaps [`t.p.deprecate`'s heuristic is wrong?](https://github.com/twisted/twisted/blob/0aef33642cbe04cac558a4aaceeb2a6a03de403b/src/twisted/python/deprecate.py#L455-L462).

I'm going to remove the suppression because it's actually useless!